docs/cost-in-cosmos.md

```diff
@@ -2,7 +2,7 @@
 
 # Determining the cost of operations in Cosmos
 
-This document should help you determine the cost of operations when Batcher is applied to Cosmos.
+This document should help you determine the cost of operations when Batcher is applied to [Azure Cosmos DB](https://azure.microsoft.com/en-us/services/cosmos-db/).
 
 Cosmos is a rate limited NoSQL database solution. This means that operations you execute against the datastore have a cost that is counted against your provisioned capacity. Specifically, when you create a Cosmos database and/or collection, you are specifying a capacity in Request Units (RU) per second. Each operation also has a cost in RU. If you exceed your allotment<sup>1</sup>, you will receive TooManyRequests errors and eventually will be cutoff with ServiceUnavailable errors.
```

Comment:
@awkwardindustries  @plasne This is the main content I had in mind for contributing back to MicrosoftDocs - e.g. 
https://docs.microsoft.com/en-us/azure/cosmos-db/optimize-cost-reads-writes#measuring-the-ru-charge-of-a-request or even a new page around best practices under parent section.

Reply:
New stories to be created. Resolving this comment.